nuxeo-tools-hooks/nxtools/hooks/services/jira_service.py

```diff
@@ -82,15 +86,23 @@ def get_issue(self, id, fields=None):
         :rtype: jira.resources.Issue
         """
         if self.__jira_client is not None:
-            return self.__jira_client.issue(id, fields)
+            try:
+                return self.__jira_client.issue(id, fields)
+            except Exception as e:
+                log.debug('get_issue: %s', e)
```

Comment:
It would rather use:
```suggestion
                log.debug('get_issue', exc_info=True)
```
It will print the full traceback, it may be useful when debugging.

Reply:
Actually i'm not sure we want the whole stacktrace here: the parsing can indeed retrieve things that are not jira issue references, and jira has its own log details holding the response if needed (see the details on the stack at https://jira.nuxeo.com/browse/NXBT-3327).
That might be valid for other logs in related PRs though.
WDYT?

Reply:
Yes, same feedback for all same stuff. But as you said, it may be too much information for nothing :)

Reply:
Improved error logs on #16, leaving this one as is as discussed :+1: